src/datepicker/types.js

```diff
@@ -231,6 +231,7 @@ export type SharedStylePropsT = {
   $isHighlighted: ?boolean,
   $isHovered: ?boolean,
   $isFocusVisible: ?boolean,
+  $month: ?boolean,
```

Comment:
should this be a `?number` type?

Reply:
Yes, it was a mistake @chasestarr. Thanks for pointing it out ☺️